apps/elasticsearch/templates/services/app/job_set_index_template.yml.j2

```diff
@@ -0,0 +1,40 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "es-index-tpl-{{ job_stamp }}"
```

Comment:
Explicit is always better than implicit. That's the zen of python :wink: 

```suggestion
  name: "es-index-template-{{ job_stamp }}"
```

Reply:
I think the fear of reaching maximum character length got me :smile: